src/components/Header/Header.tsx

```diff
@@ -1,48 +1,34 @@
 "use client";
 import NextLink from "next/link";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faBars, faUser, faXmark } from "@fortawesome/free-solid-svg-icons";
+import { faUser } from "@fortawesome/free-solid-svg-icons";
 import Link from "../Button-Link/Link/Link";
 import "./Header.css";
 import { useState } from "react";
-import Button from "../Button-Link/Button/Button";
 import { User } from "@/services/types";
+import ProfileSideNav from "../ProfileSideNav/ProfileSideNav";
 
 type Props = {
   nav?: { href: string; title: string }[];
   user?: User;
+  className?: string;
 };
 
-const Header = ({ nav, user }: Props) => {
-  const [showNav, setShowNav] = useState(false);
-
-  const displayNav = () => {
-    setShowNav(!showNav);
-  };
+const Header = ({ nav, user, className }: Props) => {
   return (
-    <header className="Header">
+    <header className={`Header ${className} `}>
       <div className="flex justify-between items-center p-4">
-        <Button
-          variant="text"
-          color="black"
-          onClick={displayNav}
-          type="button"
-          size="small"
-          className="menuButton"
-        >
-          <FontAwesomeIcon
-            icon={showNav ? faXmark : faBars}
-            className="h-5 w-5"
-          />
-        </Button>
         <NextLink href="/">
           <img src="/assets/logo.png" alt="logo" width="80" className="logo" />
         </NextLink>
         {user !== undefined ? (
-          <NextLink href="/profile">
+          <div className="user-profile">
             <FontAwesomeIcon icon={faUser} className="mr-2 h-5 w-5" />
             {user.fullName}
-          </NextLink>
+            <div className="paddingForDropDown">
```

Comment:
Why not tailwind css instead of adding a custom class name? You can add responsive styles over there as well.